tutorial/source/baseball.ipynb

```diff
@@ -0,0 +1,3238 @@
OXJXCwq0rki0aCfoN9HRyJcNtfCG+Be3mb9jw6dntfFz3PCyvY4h86kanqhGi01uY2Flv6KyjL+pUaSpXPghes6OHBqjCMDKfb1j7Cmo7H3UA/diPM3fcMlY1HHs+hIhHnRxk7+7Wf7+crDB7jz10dI5Qq84dIVAJzb08xLNnfx/SeONgx27zsxwjndTc4mxE1LLFhWcOvGHYhv1KvMkaE8ngVQlrLuLshzo4PcOw9b1e+nR7PO9SDp2fS+71s7+cS9k+vNpjkrjIXeAZ+pIkN5PYJkJFCRcqazkJqjQeIhf9mu1boQVg9cf/RV5/OBl24a1+5K79DOjGYZSGXLA9yumIU3wK2Lk2rFAgZTeZqiAdZ2JljeFuW2hw+SyhYI+qUsxqDd6ibPYnz5Bd2cGs3y830l70JfKFpcMtQL13bw2+d386n/2sPeE95WJ1ZqcXsixMnRcqmqbyjN22/bzttu286nquz4vK760qZImQyVyRcJ+X1Oc8XlrdZiO3TGGAsvy9tiZPNF+obrJwDoILbXg9ZZNscGU6XeYC7v8sp11rjfn/2mn4Onxli7pDy4PVES4QDL26I8e9wyFkG/VL14u/nEa7dw9YZO/tddT/N/7nmG85Y1cUFvi/P471zQw8mRLI8+Vz3LT7P/5CirO6off7025e5NXbtTNFvDWFTzLOzfqdt+5AtFjg+lq3oW5/Y00xIL8pPd/SilbBnK7kbhkdP7htIVUvlEOSuMRdDvIxkpL9bqH84gUpKoNLVkqETYGufY0xIt23GdGE6XtfqYLM3RICJwfChDOlekJRYqFQlm8gzpmEUtz2Iky89+08/XHivf2Q/Z8Q8R4Y+vXcerL+rlnnddRacne2lgzPKeAh7Ddu3GTpKRAHftqPRC3LtKgL+66VxiYT/vv+MJpzYESgWJHfGwMzQJrKybGz75M36+7yTN0aAjtbnRnTNj9t+iy9PyQ49U1cTtHV21oPzZzsq2yl2rFz3jBUqeRdrO0NNB7GOD6bIW9ZoNXQk6EmG+/stDZAvFiuD2ZNjYlWT38eGyjrP1aI4G+fybt/HO69aRzhV5y+Wryh5/0aYlxEP+iqwpN0PpHCdHMqzurG4smusMQBr2BLihdkuSdBVjscLzP+obzlAoqqqehd8nvGjjEn6y+4SVnl8olmQo13VsOJ1jNFuga4rXqbrGQkTeb3//jIh82vs1pd88y7R5irX6RzK0xUIVF8dkpHJanuVZWItiWWu0TIbqsyWWqRLw+2iKBJ3gVlM0SNyeWzGUzpdpoW70yfH+O3by5i8+xge/82RZXYOWoQDecOkKPvHaC+252eEy6WowlSvzFDRWVspS7n26zzm59cnvNbSdyTAffvlmfvX8AF95+ABgLYjhdJ4lTZEywwbwgyePMZrJc8+fXsW5PU0VU/Cg5KrH7AXV5Wn5kckXnEwozfLWqJNuOFEW0znvRUscz9cxFnrGS1Mk4LTk10HtrcttYzFgVQ6HAr6yC50lW7az024KuLbGxXYibFya5LmTo5wezdaVoNz4fMJ7b9jI9g9dz+9uW172WCTo54Zzl/LDp447sQRL5i1d/A/Y2Yu1PYtgzQB3ecyivkSs+0tFXTJUJOinuznieBaHT1emzbq5btMSzozl+K9n+gBcAe6gsynT68WdTTUZGnkWeo7oduDxKl8LBnc7Dahs9aFJRALkCsoJnIJ1AugTdVlLyVjonkdTtdiatnjImbDXEg3i84mzQxhKWW54xHNhbIuHWNYSZe2SBK+/ZLnz2cByYUcy1QcjeUehDoxly9Jm3dx04TJGMnnut7OpBqrIUJpXbV3GRSta+OZ2a+yjTpvtTIZLLdXtxbOnb4TVHXHWdCaI18je0AtKy1BLkhHOjOWcAsNMvlghTfS2xaYSs1g057yXnpYofp84AVGAHzxxjJd/5kFHw9fB7YtXtlJU1s5Wy37rliSJh/wcHUzZqdPBslYeAFes7XB+nmrMAqy4Rb6oeOLIgJNKPl68NVSa37mwm8FUjof29vPs8SGu+fsHeM83dzqP6zW4poaxaIlaMy2qxX7KYxal/lXV0BuhqKcT9Uq7oSCU4qO9VTwLgKs3dBLwCd963OpfqWvKkpGAY7iOD1ZP758odU21HmWqlLptSr9lHtAWD5XtuGsZi6QrA0nnPw+n8879y1qjDIzlGM1Y7YDTueK0eBZg7Vh22bnt+gJvpcDlCOZ9JCOVizPg9/HzD1wHwPYDp/n6Lw858pKWrqoZi/ZEuCxHe6CGZwFw+dp2OhJh7t5xlOs2LeGxA6cJ2IbMi4hw5fpO/vn+PYxk8mXjZL0t1ff1jzhNFxPhQNVOnt7dV7OdOjyUytGeCNvGwutZxLj36eOTmhS2mM55L0G/j2Weoq/7nunjqSNDHDg1yrolSSe4vW1VGw/s7ufImZRrZxpmaXOEYwNp4zbQ5gAAHlZJREFUCkpVyJAAV6y3jEV7PFQWz5gsOgX90OkUl6yannV25bpOmqNB/uWn+9nTN8yZsRyP7j/lTHrc3z+KCKyokvYL1iapqKzrQrNnzQynrZqsWMhPNOjHJ+OQoULlxmJVe5wfPnWcVLbgxI16angWzdEgl6xq42E7Y7Hdkzqr2xHB1I3FuGIWIrJBRG4VkXtF5H79NaXfPMtYzQRLrmP/cKYiEwpw1TZYF9psvkgmX3SCzU767ECKfn0hnCbPojUWchru6Qu3DlQNp/MV8Qov7Z7urzqNtbqxCJXVRQyO5Wqm9vp9wssv6Ob+3Sd46ace5Me7+nj7VWsqDJdmm70r3fH8gGswVMTZ6Z0ayZLJFzh4atTJt4+F/PVlKFuS04tTG8J0rlAxWnJ5W5RcQU2pknsxnPPVWNEW43lXHr/enOw+bmWi6XYe21a2Albcwn2x6WmJcmwobfcGqzxflrVEWdMRH1cdxXhY3WHNboD6abMTIRTwceO5S3nsudNEgn7++Nq1DKXzjvTz3MlRelujNYPpTk1UlZYfekCTi
```

Comment:
Unfortunately, this isn't correct, since chain information is not preserved when traces are yielded. I would suggest simply removing this part, and we can update our tutorial once chain diagnostics and any chain related refactoring is implemented.